extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/util/KuduUtils.java

```diff
@@ -11,6 +11,9 @@
 public class KuduUtils {
 
   public static final byte[] EMPTY_PARTITION_KEY = new byte[] {0};
+  public static final byte[] EMPTY_KEY = new byte[] {0};
```

Comment:
Could we combine this with `EMPTY_PARTITION_KEY` (replace `EMPTY_PARTITION_KEY` with `EMPTY_KEY`)?